certbot/CHANGELOG.md

```diff
@@ -39,7 +39,8 @@ More details about these changes can be found on our GitHub repo.
 
 ### Changed
 
-*
+* Dropped 32 bit support for the Windows beta installer
+* Windows beta installer is now distributed as "certbot-beta-installer-win_amd64.exe"
```

Comment:
Sorry one last thing: since we are changing the Python bitness, it also means that Certbot is not installed at the same location, and running this new version on an existing version may lead to inconsistent behavior.

I think we should say here that users should uninstall explicitly the old version before running the new installer.

Reply:
oh good catch! how about:

> * Windows beta installer is now distributed as "certbot-beta-installer-win_amd64.exe".
>   Users of the Windows beta should uninstall the old version before running this.

Reply:
Also, since a release has been done in the mean time, you need to move these new entries up to the master section.

Reply:
fixed